linux_os/guide/system/permissions/partitions/mount_option_dev_shm_nodev/rule.yml

```diff
@@ -30,3 +30,26 @@ references:
     cis-csc: 11,13,14,3,8,9
 
 platform: machine
+
+template:
+    name: mount_option
+    vars:
+        filesystem: tmpfs
+        filesystem@rhel6: ''
+        filesystem@sle12: ''
+        filesystem@wrlinux1019: ''
+        filesystem@wrlinux8: ''
+        mount_has_to_exist: 'no'
+        mount_has_to_exist@rhel6: 'yes'
+        mount_has_to_exist@sle12: 'yes'
+        mount_has_to_exist@wrlinux1019: 'yes'
+        mount_has_to_exist@wrlinux8: 'yes'
+        mountoption: nodev
+        mountpoint: /dev/shm
+        type: tmpfs
+        type@rhel6: ''
+        type@sle12: ''
+        type@wrlinux1019: ''
+        type@wrlinux8: ''
+    template_backends:
```

Comment:
We currently use "backends", not "template_backends".

Reply:
Fixed to `backends`.